source/AsyncRgbLedAnalyzerSettings.h

```diff
@@ -10,18 +10,33 @@ class AsyncRgbLedAnalyzerSettings : public AnalyzerSettings
 	AsyncRgbLedAnalyzerSettings();
 	virtual ~AsyncRgbLedAnalyzerSettings();
 
-	virtual bool SetSettingsFromInterfaces();
+	bool SetSettingsFromInterfaces() override;
 	void UpdateInterfacesFromSettings();
-	virtual void LoadSettings( const char* settings );
-	virtual const char* SaveSettings();
+	void LoadSettings( const char* settings ) override;
+	const char* SaveSettings() override;
 
-	
-	Channel mInputChannel;
-	U32 mBitRate;
+	enum Controller {
+		LED_WS2811,
+		LED_WS2812B,
+		LED_WS2813,
+		LED_TM1809,
+		LED_TM1804,
+		LED_UCS1903,
+		LED_LPD1886_8bit,
+		LED_LPD1886_12bit
+	};
 
+	Controller mLEDController = LED_WS2811;
```

Comment:
love initialization in the class definition, I'll check to make sure this works with all of our targets. more detail in the overall review notes.